src/main/java/com/project/ita5/answer/validator/AnswerValidator.java

```diff
@@ -0,0 +1,55 @@
+package com.project.ita5.answer.validator;
+
+
+import com.project.ita5.answer.Answer;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Arrays;
+
+public class AnswerValidator implements ConstraintValidator<Conditional, Answer> {
+//    private String[] selected;
+//    private int max;
+    private String message;
+//    private String[] values;
+    private String[] values250 = {"1", "2", "6"};
+    private String[] values450 = {"3", "4", "5"};
+    private String[] valuesYesNo = {"3", "4", "5", "6"};
+
+    @Override
+    public void initialize(Conditional constraintAnnotation) {
+//        selected = constraintAnnotation.selected();
+//        max = constraintAnnotation.max();
+        message = constraintAnnotation.message();
+
+    }
+
+    @Override
+    public boolean isValid(Answer answer, ConstraintValidatorContext context) {
+        Boolean valid = true;
+        try {
+            if (Arrays.asList(values250).contains(answer.getQuestionId())) {
+                if (answer.getAnswer().length() > 250) {
+                    valid = false;
```

Comment:
You could return false here instead of checking other conditions after finding a wrong one.

Reply:
Done